stablehlo/transforms/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ add_mlir_dialect_library(StablehloPasses
   StablehloAggressiveFolder.cpp
   StablehloAggressiveSimplification.cpp
   StablehloCanonicalizeDynamism.cpp
+  StablehloConvertToSignless.cpp
```

Comment:
Probably need this in `BUILD.bazel` as well, should just be adding the file name.